next-packages/utils/src/index.js

```diff
@@ -52,7 +52,9 @@ export const insertStyles = (
     )
     context.inserted[insertable.name] = true
 
-    if (!isBrowser) {
+    if (isBrowser) {
```

Comment:
just reassuring myself - this is inlined at build time and 4 bundles are created, right? 2 for module/main and 2 for their browser alternatives

Reply:
also - is reordering this if/else logic important in any way?

Reply:
Yes, it's inlined. The reordering isn't important, I just did it to avoid the negation.